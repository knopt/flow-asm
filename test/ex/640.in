5
8
-3.160487595326882
-9.46166501999892 1.1047102477369997 1.9407456257648033 -6.222235311841393 7.685861928980845 
3.4189638734861756 -2.8757007856190135 0.5340082807096369 -0.6232480931095914 -3.9631866459229386 
-2.5398989279733257 -4.038039234618484 2.853193841189947 -7.210520816041628 -5.531543265499391 
8.11328551075081 -7.249083657280626 3.7339691957686743 -8.65923421852389 -5.8689623907507915 
-3.053047456800561 3.4829689686056486 5.273805766608996 6.749764714793503 -8.57108597520147 
-6.322455888957967 6.590574896426148 -2.1534737066493186 4.79190106252085 -4.80076490098716 
-1.7867498686327181 0.10263439522525353 -5.363849663229471 1.472240689469892 3.572372496979737 
8.398028783255672 -0.7680933594294075 -3.393495486341928 -7.6590860916948555 6.789735343949623 

0.2264217221065863 -0.6477476678688792 5.3268210565127525 -4.126395500805289 -5.530004482809625 -3.764081536386678 9.510580942190796 -6.857843891009545 
