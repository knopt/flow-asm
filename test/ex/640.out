-108.6460872863862 39.76547802758854 24.251320132941252 -71.06978902249546 114.71974340456636 
79.87396596423031 -11.922833142605821 12.180821334446762 15.143058919652773 -28.976068239950813 
-80.87075693801441 -64.26986021960377 79.21315122796302 -114.32353220470006 -9.74309577310615 
167.68299474361837 -128.01348639229056 61.71943603748222 -181.5264767116667 -25.218075505920247 
-14.543269970354375 64.59280264977563 74.69098645634077 103.97222735037431 -119.40348892560127 
-91.62078304187045 134.68363379439532 -68.06071093136177 61.620683859321744 -105.99647250374301 
-33.06918577345884 -17.59019420151526 -91.32087936095664 68.28244323965095 58.14715517928825 
85.28668441913193 -29.2699816948935 -16.51324152908489 -57.25394356791021 79.42936751961125 
